### PR TITLE
`dl-v2` fix license deployment logging

### PR DESCRIPTION
**Pull request by aniketsinghrawat**
All the logs of license deployment in GCP was being marked as `Error` instead of it's marked log level.
This PR fixes that issue.

Also the "worker is busy" log was being logged every second when workers were busy. That has been reduced to every 60 seconds.

### CODE REVIEW

**Review by mahrsee1997 — APPROVED** [review]
LGTM!

File: weather_dl_v2/license_deployment/fetch.py [hunk 1]
```diff
@@ -163,7 +169,11 @@ def main():
             # it's concurrency_limit. When an executor is freed up, the task
             # in queue is picked and license fetches another task.
             while executor._work_queue.qsize() >= 1:
-                logger.info("Worker busy. Waiting...")
+                # To prevent flooding of this log, we log this every 60 seconds.
+                if log_count%60 == 0:
+                    logger.info("Worker busy. Waiting...")
+                # Reset log_count if it goes beyod 3600.
+                log_count = 1 if log_count > 3600 else log_count
```
Comment (mahrsee1997) [inline weather_dl_v2/license_deployment/fetch.py:176]: nit: log_count + 1

File: weather_dl_v2/license_deployment/fetch.py [hunk 2]
```diff
@@ -163,7 +169,11 @@ def main():
             # it's concurrency_limit. When an executor is freed up, the task
             # in queue is picked and license fetches another task.
             while executor._work_queue.qsize() >= 1:
-                logger.info("Worker busy. Waiting...")
+                # To prevent flooding of this log, we log this every 60 seconds.
+                if log_count%60 == 0:
+                    logger.info("Worker busy. Waiting...")
+                # Reset log_count if it goes beyod 3600.
```
Comment (mahrsee1997) [inline weather_dl_v2/license_deployment/fetch.py:175]: nit: typo 'beyond'

File: weather_dl_v2/license_deployment/fetch.py (hunk 1 above)
Comment (aniketsinghrawat) [reply weather_dl_v2/license_deployment/fetch.py]: done

File: weather_dl_v2/license_deployment/fetch.py (hunk 2 above)
Comment (aniketsinghrawat) [reply weather_dl_v2/license_deployment/fetch.py]: fixed

**Pull request merged**